addons/project/tests/test_personal_stages.py

```diff
@@ -90,6 +90,29 @@ def test_personal_stage_read_group(self):
         self.assertEqual(1, total_stage_0)
         self.assertEqual(1, total_stage_1)
 
+    def test_default_personal_stage(self):
+        user_without_stage = self.env['res.users'].create({
+            'login': 'test_no_stage',
+            'name': "Test User without stage",
+        })
+        user_with_stages = self.env['res.users'].create({
+            'login': 'test_stages',
+            'name': "Test User with stages",
+        })
+        personal_stage = self.env['project.task.type'].create({
+            'name': 'personal stage',
+            'user_id': user_with_stages.id,
+        })
+        # ensure that a user without personal stage is getting the default stages
+        self.task_1.with_user(user_without_stage).action_menu_view_all_task()
+        stages = self.env['project.task.type'].sudo().search([('user_id', '=', user_without_stage.id)])
+        self.assertEqual(len(stages), 7, "The default personal stage were not correctly created for the user")
```

Comment:
```suggestion
        self.assertEqual(len(stages), 7, "As this user had no personal stage, the default ones should have been created for him")
```
I like to give a short explanation.